modules/9_gan/test_generator.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
```

Comment:
Please rename this file to `test_gan`. Testing infrastructure tries to run a script with the same name as module.

Make sure that `chmod +x test_gan.py` is applied.